README.md

```diff
@@ -25,24 +25,24 @@ Guide.
 
 ## Installation
 
-We provide three options for installing Spine toolbox:
+We provide three options for installing Spine Toolbox:
 (i) from the [Python Package Index (PyPI)](https://pypi.org/project/spinetoolbox/),
-(ii) from stand-alone installation packages,
+(ii) using stand-alone installation packages,
 and (iii) directly from sources.
 
 ### Installation from PyPI (recommended)
 
 This is the recommended way to install Spine Toolbox and keep it updated with new releases.
 
 ---
-**NOTE:** If you have previously installed Spine Toolbox using Conda, 
-please follow the steps [here](#uninstalling-a-previous-conda-installation) to uninstall it.
+**NOTE:** If you have previously installed Spine Toolbox in a conda environment, 
+please follow the steps [here](#uninstalling-a-previous-installation-in-a-conda-environment) to uninstall it.
 
```

Comment:
The idea here is that users who installed following the 'old' instructions (currently in master) recognize their situation. I don't know if they're very much aware of what a conda environment is. I think they will remember thought that they installed "using Conda", that's why I used that formulation.